go.mod

```diff
@@ -15,7 +15,6 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/spf13/pflag v1.0.5
 	k8s.io/api v0.17.2
-	k8s.io/apiextensions-apiserver v0.17.1
```

Comment:
is this related to the documentation change?

Reply:
I don't believe so. I ran `go mod tidy` after rebasing onto master,  should I remove that change from the commit or leave it as it seems that go modules thinks it's not being used?

Reply:
probably better to leave it for another PR just for historic tracking purposes.